config/webpack.dev.js

```diff
@@ -54,8 +54,8 @@ const config = {
     overlay: true,
     open: true,
     proxy: {
-      "/container1": {
-        target: "http://localhost:8080/",
+      "/container1/*": {
+        target: "https://18.208.166.121.nip.io:8443/",
```

Comment:
Is it safe to post IP here? :)

Reply:
I can change this to use a constant that is not checked in to the repo & is ignored by .gitignore file